qa/tasks/vstart_runner.py

```diff
@@ -940,7 +940,7 @@ def run_ceph_w(self, watch_channel=None):
         if watch_channel is not None:
             args.append("--watch-channel")
             args.append(watch_channel)
-        proc = self.controller.run(args=args, wait=False, stdout=BytesIO())
+        proc = self.controller.run(args=args, wait=False, stdout=StringIO())
```

Comment:
@tchaikov 
The https://github.com/ceph/ceph/pull/34732/commits/dd4fc808a8a26ebbfae5b5e4bc3b6c99bf0f5d37 is also fixing it.

Reply:
i am not sure what i am supposed to do? this commit has not been merged. neither can i find the PR containing it.

Reply:
Sorry, I misreaded it. Thanks.